configuration/config2/SchemaTypeStringLimits.cpp

```diff
@@ -35,8 +35,8 @@ bool SchemaTypeStringLimits::doValueTest(const std::string &testValue) const
     // test patterns
     for (auto it = m_patterns.begin(); it != m_patterns.end() && isValid; ++it)
     {
-        std::regex expr ((*it).c_str());
-        isValid = std::regex_match(testValue, expr);
+        //std::regex expr ((*it).c_str());
```

Comment:
this function is now a no-op? is this inadvertent?

Reply:
There is another PR that will overwrite this code when merged. See PR 11410.